### PR TITLE
Update support tables for OpenJDK 13

### PR DESCRIPTION
**Pull request by SueChaplain**
- Add further rows for JDK 13 and 14 with appropriate notes
- Add a supported platform table for JDK 13
- Add a build system table for JDK 13

OpenJDK 12 information retained until we ship 0.17 end of October.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
@pshipton - please check the tables. I assume there are no changes to OS support or compilers, but let me know if you need any changes. Thanks!

File: docs/openj9_support.md [hunk 1]
```diff
@@ -43,18 +43,20 @@ this table over time.
...
+| v0.10.0         | September 2018      | No        | Yes (\*2)   |          |           |           |
+| v0.11.0         | October 2018        | Yes       | Yes         |          |           |           |
+| v0.12.0         | January 2019        | Yes       | Yes         |          |           |           |
+| v0.13.0         | March 2019          | No        | No          | Yes (\*2)|           |           |
+| v0.14.0         | April 2019          | Yes       | Yes         | Yes      |           |           |
+| v0.15.0         | July 2019           | Yes       | Yes         | Yes      |           |           |
+| v0.16.0         | September 2019      | No        | No          | No       | Yes (\*2) |           |
+| v0.17.0         | October 2019 (\*1)  | Yes       | Yes         | No       | Yes       |           |
+| v0.18.0         | January 2020 (\*1)  | Yes       | Yes         | No       | Yes       |           |
+| v0.19.0         | March 2020 (\*1)    | Yes       | Yes         | No       | No        | Yes (\*2) |
```
Comment (pshipton) [inline docs/openj9_support.md]: JDK8 and 11 should be No on this line. We could remove some of the earlier releases, not sure we need to keep all the history.

File: docs/openj9_support.md [hunk 2]
```diff
@@ -161,6 +163,48 @@ platform support might vary, depending on their build environment. Check the Ado
...
+| OS X 10.9.0+            |    Y     |
+
+| AIX          |  ppc64   |
+|--------------|----------|
+| AIX 7.1 TL4  |    Y     |
+| AIX 7.2      |    Y     |
+
+When public support for an operating system version ends, OpenJ9 can no longer be supported on that level.
+
+### OpenJDK 13
```
Comment (pshipton) [inline docs/openj9_support.md]: You could just change 12 -> 13 instead of adding a new jdk13 section, since 12 is obsolete now.

Comment (SueChaplain) [reply docs/openj9_support.md]: Last year we removed 10 1Q after 11 GA'd. We'll follow the same pattern this time, removing 12 1Q after 13 GAs. There will be some tidy up work to do across the docs to ensure all the icons are updated, so we'll tackle it all together in one hit.

File: docs/openj9_support.md (hunk 1 above)
Comment (SueChaplain) [reply docs/openj9_support.md]: Good spot. I changed the values and also removed 2018 rows. Will remove JDK 12 when we remove it generally from the docs in October.

**Comment by pshipton** [comment]
Pls remove the .DS_Store file from the commit.

**Comment by SueChaplain** [comment]
>Pls remove the .DS_Store file from the commit.

I *think* I might have achieved that?! Does it look ok to you @pshipton?

**Pull request merged**